dsa-solutions/lc-solutions/0000-0099/0083-remove-duplicates-from-sorted-list.md

```diff
@@ -1,121 +1,122 @@
 ---
 id: remove-duplicates-from-sorted-list
-title: Remove Duplicates from Sorted List Solution
-sidebar_label: 0083 Remove Duplicates from Sorted List
-tags:
+title: Remove Duplicates from Sorted List (LeetCode)
+difficulty: Easy
+sidebar_label: 0083-RemoveDuplicatesFromSortedList
+topics:
   - Linked List
-  - Two Pointers
-  - LeetCode
-  - Python
-  - Java
-  - C++
-description: "This is a solution to the Remove Duplicates from Sorted List problem on LeetCode."
+companies: []
```

Comment:
remove `companies: []`, `topics:  - Linked List` because I can't use it I can be used in future based on requirements but in this time I'm not using so, delete it.